packages/pwa-buildpack/lib/queries/getSchemaTypes.graphql

```diff
@@ -0,0 +1,12 @@
+# Introspect the GraphQL schema to obtain information about its types.
```

Comment:
Note that `buildpack` now has its own `queries` folder with `.graphql` files in it.

This allows us to not have to inline large GraphQL strings 🎉